scripts/makeSummary.R

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env Rscript
+### script to create a summary file of all the organism part references we created
+
+# Load required packages
+suppressMessages(library(devtools))
+devtools::install_github("YY-SONG0718/scOntoMatch")
+suppressMessages(library(scOntoMatch))
+suppressMessages(library(ontologyIndex))
+suppressMessages(library(stringr))
+
+args <- commandArgs(trailingOnly = TRUE)
+
+# read in list of UMAP.pngs we created
+sample_list <- args[1:(length(args)-1)]
+outfile <- args[length(args)]
+
+if (!file.exists(outfile)) {
+  write.table(data.frame(Accession = character(), UBERON_ID = character(), 
+                         organism_part = character(), date = character()),
+              outfile, sep = "\t", row.names = FALSE, col.names = TRUE)
```

Comment:
```suggestion
write.table(data.frame(
  Accession = character(),
  UBERON_ID = character(),
  organism_part = character(),
  date = character()
  ), 
  outfile, sep = "\t", row.names = FALSE, col.names = TRUE)
```
This makes it easier to see the column names and the values being passed to data.frame().